solrCloud_external_zooKeeper/Dockerfile

```diff
@@ -0,0 +1,34 @@
+#FROM solr:8.11.2
+FROM --platform=$BUILDPLATFORM solr:8.11 AS indexer
```

Comment:
We want to be somewhat careful about using the `--platform=$BUILDPLATFORM`. The usage there is really only if we want to create some data and then copy it to another image. Basically (let's say this is running in github actions) if we want to build both an arm and x86 example index, this let us do the indexing once on x86 and then copy the result into both an arm and x86 final image (again, see the catalog example index https://github.com/hathitrust/hathitrust_catalog_indexer/blob/main/example-index/Dockerfile for some more context)

Reply:
Done